docs/user/exports.rst

```diff
@@ -0,0 +1,90 @@
+.. |download_exercise| image:: exports/download_exercise.png
+.. |download_repos| image:: exports/download_repos.png
+.. |download_scores| image:: exports/scores.png
+.. |export_scores| image:: exports/export_scores.png
+.. |export| image:: exports/export.png
+.. |scores_navigation_bar| image:: exports/scores_navigation_bar.png
+.. |privacy_statement| image:: exports/privacy_statement.png
+.. |data_export| image:: exports/data_export.png
+
+
+Exports
+=======
+
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Overview
+--------
+Artemis offers several options to export or archive different data. The following table gives an overview of the available export options.
+
+.. list-table:: Export options
+   :widths: 100
+   :header-rows: 1
+
+   * - Export/Archive option
+   * - Archive course/exam
+   * - Export programming exercise material
+   * - Export programming exercise student repositories
+   * - Export exercise results
+   * - Export quiz questions
+   * - Export exercise submissions
+   * - Export user data
+
+Archive course/exam
+-------------------
+Export all course/exam data, including all exercises and student submissions.
+To archive a course or an exam the end date of the entity needs to be in the past.
+You can archive a course by clicking |archive_course| on the course management overview page or an exam by clicking |archive_exam| on the exam checklist page. This will create a zip file containing all relevant data.
+The creation is done asynchronously. You will receive a notification once the archive is ready to download. You can then download the archive by clicking |download_archive| on the course management overview page or the exam checklist page.
+
+Export programming exercise material
+------------------------------------
+Export the exercise material (template,solution,test, auxiliary repositories and problem statement and other relevant information such as points) of a programming exercise.
+To export the material click the |download_exercise| button on the exercise details page.
+
+
+Export programming exercise student repositories
+------------------------------------------------
+Export the student repositories (this can include the repositories for both graded and practice participations) of a programming exercise
+To export the repositories click the |export| button and then the |download_repos| button on the |download_scores| page.
+
+Export exercise results
+-----------------------------------
+Export the results of a specific exercise. This is supported for all exercise types.
+To export the results click the |export| button and then the |export_results| button on the |download_scores| page.
+
+Export quiz exercise
+--------------------
+Exports the questions and the sample solution of a quiz in JSON format
+You can export a a quiz exercise by clicking the |export_quiz| button on the exercises overview page.
+
+Export exercise submissions
```

Comment:
Nitpick: reordering the sections so that this one is before/after the ‘Export programming exercise student repositories’ ones would make sense since those do the essentially same thing with some adaptation for different exercise types.